### PR TITLE
SIT-2413 updated active support to 4.0.13

### PR DESCRIPTION
**Pull request by jonathan-fantham**

### CODE REVIEW

**Comment by cbandy** [comment]
Easy!

**Comment by jonathan-fantham** [comment]
like magic

**Comment by kelseymok** [comment]
:100:

**Comment by faraocious** [comment]
:wave: apparently i am still watching public bureaucrat PRs!

**Comment by eboyer** [comment]
@jonathan-fantham @kelseymok ~ Needs more cowbell. Let's circle back on this PR.

**Comment by jonathan-fantham** [comment]
:cow: :bell:

**Comment by jonathan-fantham** [comment]
Does that help? Or would you like me to copy paste that 300 times?

File: bureaucrat-sc.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.files       += Dir[File.join('locales', '**', '*.yml')]
 
   s.add_runtime_dependency 'i18n',          '~> 0.6'
-  s.add_runtime_dependency 'activesupport', '~> 3.2'
+  s.add_runtime_dependency 'activesupport', '<= 4.2.1'
```
Comment (cbandy) [inline bureaucrat-sc.gemspec]: :memo: Less-than-or-equal-to

**Pull request closed without merging**